### ISSUE TITLE
Update description for the setting `editor.useTabStops`

### ISSUE THREAD

**Issue by olsonpm** [microsoft/vscode#205298]
I'd like to update the description for the setting `editor.useTabStops`

Right now it reads

> Inserting and deleting whitespace follows tab stops

I've been programming for a long time and this doesn't make any sense to me.  I'd like to change it to something that's more intuitive to developers.  I can suggest more dev-friendly wording once someone explains what this setting does,

**Comment by hediet** [comment]
Feel free to just create a PR for this. Thanks!

**Comment by olsonpm** [comment]
@hediet sure, but can you tell me what the setting does so i can create a PR with updated wording ?

**Comment by BrookMaoDev** [comment]
Hi @olsonpm, I came across a helpful YouTube [video](https://youtu.be/xJzTIF8_TLE?si=nXJWRn2pzdaE4nWy) that explains the concept behind this setting. Would you like me to take a shot at improving the documentation for this issue, or are you already working on it?

**Comment by olsonpm** [comment]
I haven't - go ahead

**Comment by BrookMaoDev** [comment]
Hey @olsonpm, I've located the code related to the previous description: 

> Inserting and deleting whitespace follows tab stops.

I'm considering updating it to: 

> Inserting and deleting whitespace follows tab stops. Full tab deletion occurs upon removing indentation.

Do you think this provides a better description? I wanted to confirm with you before submitting a pull request.

**Comment by olsonpm** [comment]
Thanks @BrookMaoDev, personally "Inserting and deleting whitespace follows tab stops" feels gramatically incorrect to me.  I would go with something like

> Add a tab stop after the group of spaces when [Insert Spaces](&lt;link to setting&gt;) is set to `true`.  For example, if tab is configured to insert four spaces then the editor will insert four spaces plus a tab stop.

I'm no english expert so my verbage may be gramatically incorrect as well.  I also tend to be too wordy - however I couldn't understand the previous wording at all.  It didn't help I hadn't heard of a tab stop mixed with spaces ha.  The video you linked was very helpful.

**Comment by BrookMaoDev** [comment]
Yeah @olsonpm, I think it's a bit wordy, but I'll take your suggestion into account since I don't think my improvement was descriptive enough either.

**Comment by olsonpm** [comment]
yeah I think it will be difficult to convey this concept succinctly - my main issue is 'follows' feels incorrect.  i.e. what does it mean for "Inserting and deleting whitespace follows tabstops" ?  Examples of 'follows' being correct would be things like Bob follows Tom.  Bob follows the instructions.  Inserting and deleting whitespace can't follow tabstops - that simply doesn't make sense.

**Comment by BrookMaoDev** [comment]
Perhaps a better word is "aligns". Do you think that this makes sense?

> Inserting and deleting whitespace aligns with tab stops.

**Comment by olsonpm** [comment]
go for it.  If people comment here again I can add more context and further explanation in a PR then
